app/models/ability.rb

```diff
@@ -95,7 +95,7 @@ def school_manager_abilities(user:)
       edit_main_teacher_full_name
       edit_terms_rich_text
```

Comment:
@fitchMitch / @pierrax ; i refactored this field, forget an occurence, still a green suite. the sign of missing system test